### DISCUSSION TITLE
Where can I change the link and font colors in pagination

### DISCUSSION THREAD

**Discussion by amuijs** [templaza/astroid-framework#900] [Q&A] [answered]
Where can I change the link and font color in pagination.

![astroid pagination](https://github.com/user-attachments/assets/d3e0661e-ac7c-45b7-bf2a-1b73d729000e)

**Comment by deltapapa01** [comment, marked as answer]
Hi,
you can use custom css to change the color etc.
For example, i use a custom css file and put the css code in there;

```
.pagination {
  --bs-pagination-color: #c02e2e;
  --bs-pagination-bg: #ff0;
  --bs-pagination-border-color: #black;
}
```
After purge joomla and browser cache, the result is:

![image](https://github.com/user-attachments/assets/b090d8fa-001d-4bca-b20d-47b7df0cdcb7)

wbr,
deltapapa
